src/theme.ts

```diff
@@ -369,16 +367,14 @@ const darkColors = {
     "linear-gradient(83.46deg, #2C2C32 7.03%, #44404D 52.42%, #303038 98.77%)",
   bannerGridGradient:
     "linear-gradient(90deg, rgba(172, 182, 229, 0.08) 0%, rgba(134, 253, 232, 0.08) 100%)",
-  slider: {
-    bg: "#191919",
-    border: "#404040",
-    dot: "#A4A4A4",
-    dotActive: "#FF7324",
-    btnBg: "#404040",
-    btnColor: white,
-    btnBgDisabled: "#404040",
-    btnColorDisabled: "#737373",
-  },
+  sliderBg: "#191919",
+  sliderBorder: "#404040",
+  sliderDot: "#A4A4A4",
+  sliderDotActive: "#FF7324",
+  sliderBtnBg: "#404040",
+  sliderBtnColor: white,
+  sliderBtnBgDisabled: "#404040",
+  sliderBtnColorDisabled: "#737373",
```

Comment:
Why do we need these changes? if not used please revert

Reply:
<img width="1351" alt="Screenshot 2022-10-22 at 5 01 44 PM" src="https://user-images.githubusercontent.com/52166437/197336741-aacdf323-2cfa-4035-9165-53b62edf28e9.png">

In the previous implementation I was not able to apply the colors
Example:
```ts
bg="slider.bg"
```
This change was not reflected

I tried to reach out in [discord](https://discord.com/channels/714888181740339261/870313128242602004/1028259754360983562) but got no solution so the did the above change